js/banner-animation.js

```diff
@@ -65,8 +65,9 @@ document.addEventListener("DOMContentLoaded", function () {
         .width(bannerContainerWidth)
         .height(bannerContainerHeight)
         .globeImageUrl('//unpkg.com/three-globe/example/img/earth-blue-marble.jpg')
+        //.globeImageUrl('images/banner/DACCS-Logo-Rectangle-Waves-Gradient.png')
```

Comment:
why commented and not removed?

Reply:
This is used for overlaying the Marble logo over the globe instead of the default image.  A lot of options were discussed for replacing what there is now but nothing permanent has been put in place so I'm keeping all optional stuff there.

Reply:
Ok but we should finalize the decision about what we're keeping and removing soon